README.md

```diff
@@ -166,6 +166,91 @@ optional `--wait --timeout 15m` parameters in a deployment pipeline to see if th
 To deploy Magento to different environments (develop, staging, production) it is recommended to create a `values_*.yaml`
 for each environment and tune the resource limits and configuration values of the services.
 
+## Docker Desktop Example
+The file `values_docker.yaml` is a shorter version of `values.yaml` which contains all the changes necessary 
+to execute the chart locally on your machine and create a local cluster with the help of [Docker Desktop](https://www.docker.com/products/docker-desktop).
+The following steps will give a guide on how to achieve that:
+
+### Prerequisites
+- Install *Docker Desktop* locally (Tested with 4.10.1) and enable kubernetes in settings.
+It is also recommended to increase the available resources of Docker Desktop to 4 CPU cores and 8 GB of RAM to ensure a smooth operation. 
+Alternatively you may install *kubectl* separately, however that may be a more error-prone approach.
+- Clone this repository locally on your computer.
+
+### Step 1
+Generate a `kube-config` file which will be used for the helm deployment. Make sure to generate it
+in the same folder as the git repo. Quickest way to do so is:
+
+```
+kubectl config view --raw > <name-of-your-file>
+```
+
+### Step 2
+Have a helm environment set up. We recommend using a docker container for that in order to facilitate the usage of Helm
+and not have to worry about dependencies. However it is also possible to have a native Helm installation for our purposes.
+You may set up the docker container using the following command:
+
+```
+docker run -ti --entrypoint= -v $(pwd):/apps alpine/helm:latest sh
+```
+
+When using the native Helm installation bear in mind that the Helm version needs to be 3.9.x
+
+### Step 3
+Next we want to deploy the helm chart, but before we can do that we will need to pull the dependencies of our chart.
```

Comment:
@phoenix-henri Sounds like you are talking to someone. Be descriptive.
"Step 3: Pull chart dependencies and deploy the Helm chart."